common/djangoapps/student/views.py

```diff
@@ -349,7 +350,7 @@ def _cert_info(user, course_overview, cert_status, course_mode):  # pylint: disa
         'can_unenroll': status not in DISABLE_UNENROLL_CERT_STATES,
     }
 
-    if (status in ('generating', 'ready', 'notpassing', 'restricted', 'auditing') and
+    if (status in ('generating', 'ready', 'notpassing', 'restricted', 'auditing', 'unverified') and
```

Comment:
Seconding @mattdrayer here. CertificateStatuses are already used above. Replacing them here should be easy. Even better would be a constant assigned to this tuple, so that it doesn't need to be written out twice.